spec.bs

```diff
@@ -2792,6 +2834,23 @@ modified such that the first step of the algorithm that runs when a user-agent c
 [=traversable navigable=]'s [=system visibility state=] calls the [=Document/inclusive descendant
 navigables=] algorithm with [=inclusive-dn-unfenced|unfenced=] set to true.
 
+<h3 id=events>Events</h3>
+
+The {{Fence/notifyEvent}} method of the {{Fence}} interface introduces a new event type, <code><dfn>fencedtreeclick</dfn></code>.
+
+The <a href="https://html.spec.whatwg.org/multipage/webappapis.html#event-handlers-on-elements,-document-objects,-and-window-objects">
+event handlers on elements, Document objects, and Window objects</a> section of [[HTML]] is modified to 
+include a new row. The [=event handler=] name is <code>onfencedtreeclick</code>, and the [=event handler event type=] is
```

Comment:
```suggestion
include a new row. The [=event handler=] name is <dfn><code>onfencedtreeclick</code></dfn>, and the [=event handler event type=] is
```

This is where the definition should be. See that in HTML, the definitions are all in the table.

Reply:
Done, but I've added the definition in the table itself, see https://github.com/WICG/fenced-frame/pull/156#discussion_r1586865226